### PR TITLE
ENCD-3883-add-pmi-data

### PR DESCRIPTION
**Pull request by zoldello**
@caseylitton This contain code to add me to user.json. This relates to this feature in that it helps me with testing locally.

### CODE REVIEW

File: src/encoded/tests/data/inserts/user.json [hunk 1]
```diff
@@ -834,6 +834,21 @@
...
+        ],
+        "job_title": "Software Developer",
+        "lab": "/labs/j-michael-cherry/",
+        "last_name": "Adenekan",
+        "status": "current",
+        "submits_for": [
+            "/labs/j-michael-cherry/"
+        ],
+        "uuid": "56ad466b-8711-4b82-8f2c-8db3dc5862fd"
+    },
```
[the remark this answers is not in the document]
Comment (caseylitton) [reply src/encoded/tests/data/inserts/user.json]: @zoldello Yes, add this to a different PR.  And let's make a ticket too.  The reason, I was looking for a good intro PR we could assign every new dev.  This seems like a good one.  Then new employees can make a ticket in JIRA and  go through the whole PR-Merge process for a simple code change.  We did a similar thing on my last project, they made everyone PR themselves to a Contributors file.    

Make the name something like this.  'Add zoldello to users and Remove Glick double uuid'

**Comment by zoldello** [comment]
Removed extra dd-tag and took out test authorization code (so PR stays focused on the feature)

**Comment by zoldello** [comment]
@caseylitton I removed authorization code in test file.

**Comment by hitz** [comment]
OK, I am --- for experimentation purposes, going to dismiss @caseylitton's review.

**Pull request merged**